Adding aframe-voice-command-component

lmalave opened the pull request:
npm:  https://www.npmjs.com/package/aframe-voice-command-component
GitHub:  https://github.com/lmalave/aframe-voice-command-component
Examples on GitHub Pages:  https://lmalave.github.io/aframe-voice-command-component/examples/index.html

ngokevin commented:
Awesome! I filed some suggestion issues:

https://github.com/lmalave/aframe-voice-command-component/issues/2

https://github.com/lmalave/aframe-voice-command-component/issues/3

https://github.com/lmalave/aframe-voice-command-component/issues/4

https://github.com/lmalave/aframe-voice-command-component/issues/5

lmalave commented:
Thanks for the feedback!    I renamed the component to aframe-speech-command-component and addressed the other 3 issues that you mentioned in the new repo that is now at: 

https://github.com/lmalave/aframe-speech-command-component

ngokevin commented:
Awesome! You can also link to your GitHub pages in the GitHub description (at the top).
